src/user-management/user-management.ts

```diff
@@ -961,6 +961,23 @@ export class UserManagement {
     return `${this.workos.baseURL}/user_management/sessions/logout?session_id=${sessionId}`;
   }
 
+  async getLogoutUrlFromSessionCookie({
```

Comment:
This could use a JSDoc style comment on when you'd use this over `getLogoutUrl`.

Reply:
Adding!